plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/config/OpenLineageListenerConfig.java

```diff
@@ -83,13 +82,13 @@ public Set<QueryType> getIncludeQueryTypes()
     public OpenLineageListenerConfig setIncludeQueryTypes(List<String> includeQueryTypes)
     {
```

Comment:
I've added a support to Airlift to support direct `List<OpenLineageTrinoFacet> injection`